### ISSUE TITLE
Training procedure is unclear

### ISSUE THREAD

**Issue by mikeolasz** [tensorboy/pytorch_Realtime_Multi-Person_Pose_Estimation#13]
Dear tensorboy,

I'm having a hard time figuring out the training steps in the original repository. I see that you managed to port it to pytorch and I'm also trying to reimplement the pose machine in python using tensorflow.

At the moment I'm not interested in PAF estimation, I only want to use the heatmaps but I'm just unable to figure out the preprocessing and the training steps. I'd be glad if you could aid me in any direction.

I managed to generate the masks for the images, also the ground truth heatmaps (both are the sizes of the original image from MSCOCO). I'm wondering how did they or how did you feed the images to the net, because the training images are not the same in size and I can't find the process that does this (correct me if I'm wrong, but the network uses one fix input size).

Also it is unclear to me whether the stage's output is resized to the original size or the groundtruth heatmaps and the masks are scaled down to match the output.

Thank you in advance!

**Comment by tensorboy** [comment]
Hi, Mikeolasz,

I'm glad you also interested in this task.

The code about in here will help you a lot: https://github.com/raymon-tian/keras_Realtime_Multi-Person_Pose_Estimation/blob/master/train/COCOLmdb.py.

The images feed in the network is 368x368, and the size of the output is 46x46.

The stage's output is 46x46 and didn't resize back to 368x368. Masks also scaled down to 46x46.

Best,
Wangpeng

**Comment by tensorboy** [comment]
Try `train.py`
